packages/php-config-printer/src/NodeTraverser/ImportFullyQualifiedNamesNodeTraverser.php

```diff
@@ -27,68 +28,74 @@ public function __construct(
      */
     public function traverseNodes(array $nodes): array
     {
-        $nameImports = $this->collectNameImportsFromNodes($nodes);
-        if ($nameImports === []) {
-            return $nodes;
-        }
+        $this->collectNameImportsFromNodes($nodes);
```

Comment:
This pattern now moved passing explicit variable assign to hidden service logic.
These are harder to debug. Please keep it as it is, to make the code readable

Reply:
Agreed. I refactored the code. I hope you like this better.